streamer/src/nonblocking/quic.rs

```diff
@@ -492,7 +496,7 @@ pub fn get_remote_pubkey(connection: &Connection) -> Option<Pubkey> {
         .and_then(get_pubkey_from_tls_certificate)
 }
 
-fn get_connection_stake(
+pub fn get_connection_stake(
```

Comment:
pub(crate) should be enough?

Reply:
Does not matter much since we are not maintaining public API anymore